source/engine/physics2/jolt/ConstraintManager.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "ConstraintFactory.h"
+#include "ncengine/ecs/Entity.h"
+#include "ncengine/physics/Constraints.h"
+#include "ncengine/utility/SparseMap.h"
+
+#include <span>
+#include <vector>
+
+namespace JPH
+{
+class Constraint;
+class PhysicsSystem;
+} // namespace JPH
+
+namespace nc::physics
+{
+// Pair of entity indices associated with a constraint
+struct ConstraintPair
+{
+    uint32_t ownerId = UINT32_MAX; // Index of the Entity the constraint was added to
```

Comment:
weren't we on team `std::numeric_limits<uint32_t>::max()`?

Reply:
Yes, I see your `std::numeric_limits::max()` and I raise you an `Entity::NullIndex`.